helpers/utils.js

```diff
@@ -1,3 +1,14 @@
 export const getAcrFromIdToken = idToken => (
   JSON.parse(Buffer.from(idToken.split('.')[1], 'base64').toString('utf8')).acr
 );
+
+/**
+ * @description regex that test the Error Description in URL
+ */
+// eslint-disable-next-line no-useless-escape
+export const QUERY_ERROR_REGEX = /^[\+&=#@;\,\w\\'áàâäãåçéèêëíìîïñóòôöõúùûüýÿæœÁÀÂÄÃÅÇÉÈÊËÍÌÎÏÑÓÒÔÖÕÚÙÛÜÝŸÆŒ. \t-]{1,500}$/;
```

Comment:
is it exhaustive ? how did you build that ? where did you find that ? how can I update it ?

Put explanations in a comment.

Reply:
I try

Reply:
Doc clear